Makefile

```diff
@@ -2,6 +2,7 @@ BUILD_DIR=$(shell pwd)/build
 HANDLE_USER_DATA=$(shell base64 -w 0 scripts/handle-user-data)
 CERT_GEN=$(shell base64 -w 0 scripts/cert-gen.sh)
 VERSION=1.0.0-alpha.1
+TODAY=$(shell date +"%d-%m-%Y-%H-%M-%S")
```

Comment:
Two things really, imo it would be nice to take two potential other pieces into consideration. The sha (or at least the first part) of the HEAD commit and a potentially defined environment variable. CI systems inject a build id into the environment. We can assign this id to a variable of our liking during the build. The Makefile then checks for this variable and adds it to the build_id. If not set we have the possibility to leave it out or add a dummy, eg 'local' indicating that this is a build coming from a local dev machine.